### PR TITLE
Automatically published release

### PR DESCRIPTION
**Pull request by FeirenK**
https://github.com/iotexproject/iotex-core/issues/1010
1, circleci add Environment Variables --> GITHUB_TOKEN ( Write Privilege)
2, cp "circleci_github_release.sh"  yout  local computer  -->  change CIRCLE_TOKEN="<YOUR_ACCESS_TOKEN>"
3,  Use:   sh circleci_github_release.sh -v "v0.5.1-rc4" [-b "branch"]

### CODE REVIEW

**Comment by zjshen14** [comment]
awesome, thanks! @FeirenK

File: .circleci/circleci_github_release.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+USER="iotexproject"
+PROJECT="iotex-core"
+BRANCH="master"
```
Comment (zjshen14) [inline .circleci/circleci_github_release.sh]: we need to have option to specify the branch

File: .circleci/circleci_github_release.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+if [ "$VERSION" = "0" ]; then
+    echo "Usage: $0 -v <version> [-b <branch>]"
+    exit 1
+fi
+
+CIRCLEJOBS="nightly_bin_build_docker nightly_bin_build_darwin"
+
+
+for CIRCLE_JOB in ${CIRCLEJOBS} ; do
+   curl -d "build_parameters[CIRCLE_JOB]=${CIRCLE_JOB}" -d "build_parameters[VERSION]=$VERSION" "https://circleci.com/api/v1.1/project/github/${USER}/${PROJECT}/tree/${BRANCH}?circle-token=${CIRCLE_TOKEN}"
```
Comment (zjshen14) [inline .circleci/circleci_github_release.sh]: Say if i want to build the binaries for v0.5.0. Is this code going to make sure we use the code tagged with v0.5.0 to build them?

Comment (FeirenK) [reply .circleci/circleci_github_release.sh]: if  circleci_github_release.sh -v "version"  add -b "v.5.0"  the code tagged with v0.5.0 to build

**Pull request closed without merging**